### PR TITLE
Sealed bags and crates of salvage

### PR DESCRIPTION
**Pull request by bobigor377**
ready for review

This PR contains the sealed bags of salvage, sealed crates of salvage and their corresponding wk10 bags of salvage. These items are sold by various alt vendors. 

https://asheron.fandom.com/wiki/Sealed_Bag_of_Salvaged_Brass

https://asheron.fandom.com/wiki/Sealed_crate_of_Salvaged_Red_Garnet

### CODE REVIEW

**Comment by ziang4891** [comment]
Don't know why review was posted twice...

**Comment by Mag-nus** [comment]
If this is no longer WIP and ready for release, please edit the title to remove the WIP tag.

File: Database/Patches/2007-10-Masked Memories/9 WeenieDefaults/CraftTool/TinkeringMaterial/36570 Salvaged Brass.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+     , (36570, 150,        103) /* HookPlacement - Hook */
+     , (36570, 151,          9) /* HookType - Floor, Yard */
+     , (36570, 170,         10) /* NumItemsInMaterial */;
+
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (36570,  22, True ) /* Inscribable */
+     , (36570,  23, True ) /* DestroyOnSell */;
+
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (36570,   1, 'Salvaged Brass') /* Name */
```
Comment (ziang4891) [inline Database/Patches/2007-10-Masked Memories/9 WeenieDefaults/CraftTool/TinkeringMaterial/36570 Salvaged Brass.sql]: Should just be named 'Salvage', as the client will insert the material name before the item's string name.

File: Database/Patches/2009-10-GearsOfChange/9 WeenieDefaults/CraftTool/TinkeringMaterial/70736 Salvaged Sapphire.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+     , (70736, 150,        103) /* HookPlacement - Hook */
+     , (70736, 151,          9) /* HookType - Floor, Yard */
+     , (70736, 170,         10) /* NumItemsInMaterial */;
+
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (70736,  22, True ) /* Inscribable */
+     , (70736,  23, True ) /* DestroyOnSell */;
+
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (70736,   1, 'Salvaged Sapphire') /* Name */
```
Comment (ziang4891) [inline Database/Patches/2009-10-GearsOfChange/9 WeenieDefaults/CraftTool/TinkeringMaterial/70736 Salvaged Sapphire.sql]: As with previous, should just be named 'Salvage'

**Pull request merged**